packages/features/form-builder/utils.ts

```diff
@@ -29,8 +29,9 @@ export const getFullName = (name: string | { firstName: string; lastName?: strin
   return nameString;
 };
 
-function getFirstAndLastName(value: string | Record<"firstName" | "lastName", string>) {
+function getFirstAndLastName(value: string | Record<"firstName" | "lastName", string> | undefined) {
   let newValue: Record<"firstName" | "lastName", string>;
+  value = value || "";
```

Comment:
Allows form to be rendered with `firstAndLastName` variant without the need of schema parsing to be done.